feathr_project/feathrcli/data/feathr_user_workspace/nyc_driver_demo.ipynb

```diff
@@ -144,401 +144,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
```

Comment:
It's hard to check in this file so maybe let's do a separate checkin for this file (since it's mainly for examples etc.)